lib/widgets/show_snackbar.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+void showSnackbar({
```

Comment:
@irfanSilkron kenapa create widget ni tapi tak pakai pun?

Reply:
done pakai haha, confuse dengan widget ScaffoldMessenger.of(context).showSnackBar(snackBar); , nama function sama.